Unix/tests/http/test_httpclient.cpp

```diff
@@ -141,6 +146,33 @@ void _StrandTestClientPost( _In_ Strand* self_, _In_ Message* msg )
     else
         s_charset = "";
 
+    if (request->headers->authorization)
```

Comment:
_StrandTestClientPost has a request which is coming from the server, after parsing of the http headers. I don't have access to the original auth header. But the password and username members of the HttpRequestMsg struct which I do get are busted and parsed exactly as the server is seeing them, so the work (which I actually built but couldn't use) wasn't necessary.

Of course that is only true about Basic.  For Nego we would have to figure something else out, but this isn't about Nego today